docs/LTS-strategy.md

```diff
@@ -0,0 +1,38 @@
+# Express LTS Strategy
+
+## Goals
+
+* Maintain strict Node.js version support within a single Express major release.
+* Provide good ergonomics and developer experience for Express maintainers
+* Provide good experience for users who want to keep their Express projects that up-to-date
+* Avoid stagnation and blocking innovation that is good for the end users
```

Comment:
I think folks might read this as "in order of priority". I know that is not intended, but I wonder if we should move "Provide good experience for users who want to keep their Express projects that up-to-date" to the top?

Reply:
addressed